app/src/main/assets/fido2_privileged_community.json

```
@@ -82,6 +82,15 @@
           {
             "build": "release",
             "cert_fingerprint_sha256": "06:66:53:58:EF:D8:BA:05:BE:23:6A:47:A1:2C:B0:95:8D:7D:75:DD:93:9D:77:C2:B3:1F:53:98:53:7E:BD:C5"
+          },
```

format broken, there shouldn't be a comma here, and it should close the signatures array, then the info dictionary, then the main array

```suggestion
          }
        ]
      }
    },
```